src/Generation/Generator/Renderer/Public/ReturnType/From/Utf8StringReturnType.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Generator.Model;
```

Comment:
- Please rename the class to `String` as it can handle both cases: `GirModel.Utf8String` and `GirModel.PlatformString` if it just supports `GirModel.String`.
- Support `GirModel.String` instead of `GirModel.PlatformString`
- Remove file "PlatformString" / "PlatformStringReturnType"